src/front/app/common/components/IconButton/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import ImmutablePropTypes from 'react-immutable-proptypes';
+
+const iconButton = ({
+    title,
+    iconName,
+    clickHandler,
+    iconSize = 24,
+    iconClassName = '',
+    buttonClassName = '',
+    buttonType = 'button'
+    }) => (
+    <button type={buttonType} onClick={clickHandler} className={`icon-button  ${buttonClassName}`}>
+        <i className={`icon icon_size-${iconSize} ${iconClassName}`}>{iconName}</i>
+        {title}
+    </button>
+);
+
+iconButton.propTypes = {
+    title: ImmutablePropTypes.string,
```

Comment:
I would use immutable prop types only for immutable data structures for better semantics